packages/build/src/plugins_core/build_command.js

```diff
@@ -38,6 +38,9 @@ const coreCommand = async function ({
     handleBuildCommandOutput(buildCommandOutput, logs)
     return {}
   } catch (error) {
+    // The child process `error.message` includes stderr and stdout output which most of the times contains duplicate
+    // information. We rely on `error.shortMessage` instead.
+    error.message = error.shortMessage
```

Comment:
Related with https://github.com/sindresorhus/execa/pull/397 which I imagine you're familiar with @ehmicky 😅

Keeping the current behaviour resulted in extra output in our snapshots, usually with the same kind of information reproduced twice. Wondering what you think?

Reply:
Yes, this makes sense :+1: 
I answered below at https://github.com/netlify/build/pull/3440#discussion_r687754132